pkg/integrations/otelcol/snippet-linux.sh

```
@@ -0,0 +1,203 @@
+#!/bin/bash
+set -euo pipefail
+
+PROXY_URL=""
+if ! curl -sf --connect-timeout 5 --head {{.Endpoint}} > /dev/null 2>&1; then
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)